libchisel/Cargo.toml

```diff
@@ -7,4 +7,6 @@ repository = "https://github.com/wasmx/wasm-chisel"
 homepage = "https://github.com/wasmx/wasm-chisel"
 
 [dependencies]
+log = "0.3"
```

Comment:
Where is this used?

Reply:
Was used up until 077b7c411b7b6fa60b80a7d7b00b64e8d2abb005